app/models/chargeable_field.rb

```diff
@@ -21,9 +21,9 @@ class ChargeableField < ApplicationRecord
 
   def measure(consumption, options)
     return 1.0 if fixed?
-    return 0 if consumption.none?(metric)
     return consumption.send(options.method_for_allocated_metrics, metric) if allocated?
     return consumption.avg(metric) if used?
+    return 0 if consumption.none?(metric)
```

Comment:
@lpichler im not sure if i should remove this line entirely. `consumption.none?(metric)` could be true for a container for one of the allocated fields while the actual field is calculated differently through `current_value`

Reply:
👍